### PR TITLE
fixing rpc callbacks

### PR DESCRIPTION
**Pull request by chrisnojima**
moved dev build to staging
updated deps to rc 23.1
fixed text onPress

@keybase/react-hackers

### CODE REVIEW

File: react-native/ios/Keybase/Engine.m [hunk 1]
```diff
@@ -54,7 +54,7 @@ - (void)startReadLoop {
     for(;;) {
       NSString * data = GoKeybaseReadB64();
       if(data) {
-        [self.bridge.eventDispatcher sendDeviceEventWithName:eventName body:data];
+        [self.bridge.eventDispatcher sendAppEventWithName:eventName body:data];
```
Comment (chrisnojima) [inline react-native/ios/Keybase/Engine.m]: so they changed this back and forth so thats what broke rpcs not coming back

File: shared/common-adapters/text.native.js [hunk 2]
```diff
@@ -138,14 +138,14 @@ export default class Text extends Component {
           contentEditable
           onKeyUp={this.props.onKeyUp}
           onKeyDown={this.props.onKeyDown}
-          onClick={this.props.onClick}/>)
+          onPress={this.props.onClick}/>)
```
Comment (chrisnojima) [inline shared/common-adapters/text.native.js]: Text natively supports onPress for us

File: shared/login/signup/invite-code.render.native.js [hunk 3]
```diff
@@ -32,7 +32,8 @@ export default class Render extends Component {
         <Input autoFocus style={styles.input} hintText='goddess brown result reject' value={this.state.inviteCode} errorText={this.props.inviteCodeErrorText} onEnterKeyDown={submitInviteCode} onChangeText={inviteCode => this.setState({inviteCode})}/>
         <Button style={styles.button} waiting={this.props.waiting} type='Primary' label='Continue' onClick={submitInviteCode} disabled={!this.state.inviteCode}/>
         <Text style={styles.text} type='BodySmall'>Not invited?</Text>
-        <Text style={{flex: 1}} type='BodySecondaryLink' onClick={this.props.onRequestInvite}>Request an invite</Text>
+        <Text type='BodySecondaryLink' onClick={this.props.onRequestInvite}>Request an invite</Text>
+        <Box style={{flex: 1}}/>
```
Comment (chrisnojima) [inline shared/login/signup/invite-code.render.native.js]: just have a box grow and not text

**Comment by MarcoPolo** [comment]
:shipit:

**Comment by cjb** [comment]
:thumbsup: tested

**Pull request merged**